tests/MongoDB.Driver.Tests/Linq/Translators/PredicateTranslatorTests.cs

```diff
@@ -585,7 +697,7 @@ public void ClassEquals()
             Assert(
                 x => x.C == new C { D = "Dexter" },
                 0,
-                "{C: {D: 'Dexter', E: null, S: null, X: null}}");
+                "{ C : { Ids : null, D : 'Dexter', E : null, S : null, X : null } }");
```

Comment:
Why is this different? Were we generating an incorrect query before?

Reply:
Nope, I just added new `Ids` collection to already existing class. And it gave the new node to the generated query for this test: 
    
     `Ids : null`

Reply:
Oh... that's in a completely different file. I did not see it.